### DISCUSSION TITLE
Passive mobs like chickens are not dying to wither explosions.

### DISCUSSION THREAD

**Discussion by Tryp0xd** [TownyAdvanced/Towny#6076] [General]
Hi, I'm in a server with this plugin installed, I threw a couple of eggs to have baby chickens spawned and spawned a wither to get wither roses, however upon the initial wither explosion, I haven't managed to get the chickens killed for the drop. I tried enabling explosions etc to see if that was the issue, however its still not killing them.

Here's a small gif to summarize the issue : https://gyazo.com/d56821a86772f847cd3d8c4ab2b717d4

Please let me know if there's a command I missed to use to enable the wither's explosion to kill the chickens.

**Comment by LlmDl** [comment]
The config has this option:
```
  # Protect living entities within a town's boundaries from being killed by players or mobs.
  mob_types: Animals,WaterMob,NPC,Snowman,ArmorStand,Villager,Hanging
  ```

Chickens are considered Animals and by default are protected from other mobs.
